README.md

```diff
@@ -1,89 +1,114 @@
-5. Set up the local env:
 
-- Install & activate hermit
-  ```zsh
-  curl -fsSL https://github.com/cashapp/hermit/releases/download/stable/install.sh | /bin/bash
-  ```
-	```zsh
-  . ./bin/activate-hermit
-  ```
-- Run setup env recipe
+1. We use [Hermit](https://cashapp.github.io/hermit/usage/get-started/) to keep all our tooling in check. Install it
+   with the following command:
+    ```zsh
+    curl -fsSL https://github.com/cashapp/hermit/releases/download/stable/install.sh | /bin/bash
+    . ./bin/activate-hermit
+    ```
+
+   > **Note**
+   This will download and install hermit into `~/bin`. You should add this to your `$PATH` if it isn't already.
+
+   It is also recommended to add hermit's [shell integration](https://cashapp.github.io/hermit/usage/shell/)
+
+2. Elastic stack running locally, preferably using [Elastic-Package](https://github.com/elastic/elastic-package) (you
+   may need to [authenticate](https://docker-auth.elastic.co/github_auth))
+   For example, spinning up 8.5.0 stack locally:
+
+    ```zsh
+    eval "$(elastic-package stack shellinit)" # load stack environment variables
+    elastic-package stack up --version 8.5.0 -v -d
+    ```
+
+- _optional:_ Create local kind cluster to test against
   ```zsh
-  just setup-env
+  just create-kind-cluster
+  just elastic-stack-connect-kind # connect it to local elastic stack
   ```
 
+# Deploying Cloudbeat
 
->**Note**
->This will download and install hermit into `~/bin`. You should add this to your `$PATH` if it isn't already. Also consider to review documentation for automatic shell & IDE integration for your setup of choice.
+## Running Cloudbeat as a process
 
+### Unmanaged Kubernetes (Vanilla)
 
-## Running Cloudbeat
-Load the elastic stack environment variables.
-```zsh
-eval "$(elastic-package stack shellinit)"
-```
-
-### Kubernetes Vanilla
-Build & deploy cloudbeat:
+Build and deploying cloudbeat into your local kind cluster:
 
 ```zsh
 just build-deploy-cloudbeat
 ```
 
 ### Amazon Elastic Kubernetes Service (EKS)
+
 Export AWS creds as env vars, kustomize will use these to populate your cloudbeat deployment.
+
 ```zsh
-$ export AWS_ACCESS_KEY="<YOUR_AWS_KEY>" AWS_SECRET_ACCESS_KEY="<YOUR_AWS_SECRET>"
+export AWS_ACCESS_KEY="<YOUR_AWS_KEY>"
+export AWS_SECRET_ACCESS_KEY="<YOUR_AWS_SECRET>"
 ```
 
 Set your default cluster to your EKS cluster
+
 ```zsh
- kubectl config use-context your-eks-cluster
+kubectl config use-context {your-eks-cluster}
 ```
 
 Deploy cloudbeat on your EKS cluster
+
 ```zsh
 just deploy-eks-cloudbeat
 ````
+
 ### Advanced
 
-If you need to change the default values in the configuration(ES_HOST, ES_PORT, ES_USERNAME, ES_PASSWORD), you can
+If you need to change the default values in the configuration(`ES_HOST`, `ES_PORT`, `ES_USERNAME`, `ES_PASSWORD`), you
+can
 also create the deployment file yourself.
 
 Vanilla
```

Comment:
use consistent names: `Vanilla` -> `Unmanaged Kubernetes` (btw: i think it's `Self Managed` now)